tests/aci-e2e/e2e-aci_test.go

```diff
@@ -434,14 +434,7 @@ func TestContainerRunAttached(t *testing.T) {
 		endpoint = fmt.Sprintf("http://%s:%d", fqdn, port.HostPort)
 
 		assert.Assert(t, !strings.Contains(followLogsProcess.Stdout(), "/test"))
-		checkRequest := func(t poll.LogT) poll.Result {
-			r, _ := http.Get(endpoint + "/test")
-			if r != nil && r.StatusCode == http.StatusNotFound {
-				return poll.Success()
-			}
-			return poll.Continue("waiting for container to serve request")
-		}
-		poll.WaitOn(t, checkRequest, poll.WithDelay(1*time.Second), poll.WithTimeout(60*time.Second))
+		HTTPGetWithRetry(t, endpoint+"/test", http.StatusNotFound, 2*time.Second, 60*time.Second)
```

Comment:
nit:
```suggestion
		HTTPGetWithRetry(t, endpoint+"/test", http.StatusNotFound, 2*time.Second, time.Minute)
```

Reply:
Ah yes, we could use time.Minute ; but to revisit everwhere to be consistent, we have several duration set a 60 secs, or more (120, ...). I'll leave it as it is here for consistency with the rest